backbone/src/main/res/layout/rsb_view_submitbar.xml

```diff
@@ -1,56 +1,65 @@
 <?xml version="1.0" encoding="utf-8"?><!-- Change to LinearLayout for preview to work -->
 
-<merge xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Why remove the `<merge`? That was the important bit of the question.

Reply:
I fix this sorry